core-bundle/src/Migration/Version411/RemoveJsHighlightMigration.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Migration\Version411;
+
+use Contao\CoreBundle\Migration\AbstractMigration;
+use Contao\CoreBundle\Migration\MigrationResult;
+use Contao\StringUtil;
+use Doctrine\DBAL\Connection;
+
+/**
+ * @internal
+ */
+class RemoveJsHighlightMigration extends AbstractMigration
+{
+    /**
+     * @var Connection
+     */
+    private $connection;
+
+    public function __construct(Connection $connection)
+    {
+        $this->connection = $connection;
+    }
+
+    public function shouldRun(): bool
+    {
+        $schemaManager = $this->connection->getSchemaManager();
+
+        if (!$schemaManager->tablesExist(['tl_layout'])) {
+            return false;
+        }
+
+        $columns = $schemaManager->listTableColumns('tl_layout');
+
+        if (!isset($columns['scripts'])) {
+            return false;
+        }
+
+        $layout = $this->connection->fetchAssociative("
+            SELECT
+                COUNT(*) AS cnt
+            FROM
+                tl_layout
+            WHERE
+                scripts LIKE '%js_highlight%'
+        ");
+
+        return $layout['cnt'] > 0;
```

Comment:
I’d rename `cnt` to `count` or use this code instead 🤓:

```suggestion
        $count = (int) $this->connection->fetchOne("
            SELECT
                COUNT(*)
            FROM
                tl_layout
            WHERE
                scripts LIKE '%js_highlight%'
        ");

        return $count > 0;
```

Reply:
Changed in de3e3c299969f5d8bcf9d2b180612e0e9b4b7914.